lib/win-rebase.js

```diff
@@ -1,8 +1,17 @@
 var re = /^\s*("*)([^"]*?\b(?:node|iojs)(?:\.exe)?)("*)( |$)/
+var npmre = /^\s*("*)([^"]*?\b(?:npm))("*)( |$)/
+var path_ = require('path')
+if (path_.win32) path_ = path_.win32
 
-module.exports = function (path, rebase) {
+module.exports = function (path, rebase, whichOrUndefined) {
```

Comment:
that's fair. I'm fine with letting this go. we could also look at pulling in a test helper for mocking requires (that might be overkill though).